doc/modules/core.rst

```diff
@@ -92,6 +92,11 @@ with 16 channels:
     timestamps = np.arange(num_samples) / sampling_frequency + 300
     recording.set_times(timestamps, segment_index=0)
 
+**Note**:
+Raw data formats often store data as integer values for memory efficiency. To give these integers meaningful physical units, you can apply a gain and an offset.
```

Comment:
This is Ok, maybe we can exptlictly say that they are transformed to mV. That is, those are the meaningful physical units.

Reply:
Good point, although it's actually uV, which just goes to show how important it is for us to actually specify the units we are converting to!